PIDs:
  X:
    kp: 25.0
    kd: 15.0
    ki: 0.0
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Y:
    kp: -25.0
    kd: -15.0
    ki: -0.0
    minOutput: -10.0
    maxOutput: 10.0
    integratorMin: -0.1
    integratorMax: 0.1
  Z:
    kp: 5000.0
    kd: 6400.0
    ki: 1000.0
    minOutput: 10000.0
    maxOutput: 60000.0
    integratorMin: -1000.0
    integratorMax: 1000.0
  Yaw:
    kp: -200.0
    kd: -20.0
    ki: 0.0
    minOutput: -200.0
    maxOutput: 200.0
    integratorMin: 0.0
    integratorMax: 0.0